src/views/dashboard/main/InstanceMenu.vue

```diff
@@ -86,82 +97,172 @@
         </tbody>
       </v-simple-table>
     </base-material-card>
-    <v-alert
-      v-model="alert.show"
-      dense
-      :type="alert.type"
-      dismissible
-    >
-      "{{ alert.text }}"
-    </v-alert>
     <div class="py-3" />
 
+    <test-modal
+      v-model="showEditModal"
+      :fields="modalList"
+      :title="modalTitle"
+      :buttonText="modalButtonText"
+      @submit-form="modalFunction"
+    />
+
+    <v-snackbar
+      v-model="snack.show"
+      :timeout="timeout"
+    >
+      {{ snack.text }}
+
+      <template v-slot:action="{ attrs }">
+        <v-btn
+          :color="snack.color"
+          text
+          v-bind="attrs"
+          @click="snack.show=false"
+        >
+          Close
+        </v-btn>
+      </template>
+    </v-snackbar>
   </v-container>
 </template>
 
 <script>
   import moment from 'moment'
+  import download from 'downloadjs'
   import API from '../../../api/index'
+  import TestModal from '../pages/EditModal'
```

Comment:
maybe we should not call it `TestModal`, but `EditModal` or something?

Reply:
Done.